.circleci/continue_config.yml

```diff
@@ -634,7 +634,7 @@ workflows:
       - build_kedro:
           matrix:
             parameters:
-              python_version: ["3.7", "3.8", "3.9", "3.10"]
+              python_version: ["3.7", "3.8"]
```

Comment:
This is temporary, will have to be put back after release. The first time the build will fail because e2e tests will fail on 3.9 and 3.10 (kedro-telemetry doesn't exist yet). We want to be able to `publish_kedro` despite these failures.